zebrad/src/application.rs

```diff
@@ -43,6 +43,10 @@ pub struct ZebradApp {
     state: application::State<Self>,
 }
 
+impl ZebradApp {
+    pub const GIT_COMMIT: &'static str = env!("VERGEN_SHA_SHORT");
```

Comment:
```
env!("SHORT_SHA") || env!("VERGEN_SHA_SHORT");
```

or something like that

Reply:
fixed in https://github.com/ZcashFoundation/zebra/pull/1442/files hopefully